course_contents/2_intro_to_python/notes/01. Conditions.md

```diff
@@ -0,0 +1,101 @@
+### The `if` statement
+- the `if` statement is used to check if a conditional is `True`
+- syntax for writing the `if` statement:
+    1. use the `if` keyword at the beginning of the statement
```

Comment:
I would say that there's the `if statement`, but I the`statement` alone. is what comes after the condition. How about something like:

> Use the `if` keyword followed by a condition, a colon and the statement itself:
```python
winning_number = 51
user_number = int(input("Enter a number: ")

if  winning number == user_number:
  print("You won!")
```

Reply:
Will change this, I mixed up the terms "statement" and "expression", thanks to my brain literally translating stuff